### DISCUSSION TITLE
Ungoogled Chromium Android

### DISCUSSION THREAD

**Discussion by ghost** [Eloston/ungoogled-chromium#1989] [General]
Ungoogled Chromium Android Github page archieved, so it means ungoogled chromium android will depreciated?

**Comment by networkException** [comment]
It means that nobody is maintaining it at the moment, although maybe it should be unarchived so that people can submit patches

**Reply by networkException** [reply]
I've unarchived the android and fedora packaging repositories, they are still unmaintained but at least people could contribute or file issues

**Reply by wchen342** [reply]
Oh wait it was you :D I thought Github glitched and archived them again.
